### YamlMime:JavaType
uid: "com.azure.resourcemanager.databricks.models.ErrorInfoException"
fullName: "com.azure.resourcemanager.databricks.models.ErrorInfoException"
name: "ErrorInfoException"
nameWithType: "ErrorInfoException"
summary: "Exception thrown for an invalid response with Error<wbr>Info information."
inheritances:
- "<xref href=\"java.lang.Object?displayProperty=fullName\" data-throw-if-not-resolved=\"False\" />"
- "<xref href=\"java.lang.Throwable?displayProperty=fullName\" data-throw-if-not-resolved=\"False\" />"
- "<xref href=\"java.lang.Exception?displayProperty=fullName\" data-throw-if-not-resolved=\"False\" />"
- "<xref href=\"java.lang.RuntimeException?displayProperty=fullName\" data-throw-if-not-resolved=\"False\" />"
- "<xref href=\"com.azure.core.exception.AzureException?displayProperty=fullName\" data-throw-if-not-resolved=\"False\" />"
- "<xref href=\"com.azure.core.exception.HttpResponseException?displayProperty=fullName\" data-throw-if-not-resolved=\"False\" />"
- "<xref href=\"com.azure.core.management.exception.ManagementException?displayProperty=fullName\" data-throw-if-not-resolved=\"False\" />"
inheritedClassMethods:
- classRef: "<xref href=\"com.azure.core.exception.HttpResponseException?alt=com.azure.core.exception.HttpResponseException&text=HttpResponseException\" data-throw-if-not-resolved=\"False\" />"
  methodsRef:
  - "<xref href=\"com.azure.core.exception.HttpResponseException.getResponse()?alt=com.azure.core.exception.HttpResponseException.getResponse&text=getResponse\" data-throw-if-not-resolved=\"False\" />"
- classRef: "<xref href=\"com.azure.core.management.exception.ManagementException?alt=com.azure.core.management.exception.ManagementException&text=ManagementException\" data-throw-if-not-resolved=\"False\" />"
  methodsRef:
  - "<xref href=\"com.azure.core.management.exception.ManagementException.getValue()?alt=com.azure.core.management.exception.ManagementException.getValue&text=getValue\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"com.azure.core.management.exception.ManagementException.toString()?alt=com.azure.core.management.exception.ManagementException.toString&text=toString\" data-throw-if-not-resolved=\"False\" />"
- classRef: "<xref href=\"java.lang.Object?alt=java.lang.Object&text=Object\" data-throw-if-not-resolved=\"False\" />"
  methodsRef:
  - "<xref href=\"java.lang.Object.clone()?alt=java.lang.Object.clone&text=clone\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"java.lang.Object.equals(java.lang.Object)?alt=java.lang.Object.equals&text=equals\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"java.lang.Object.finalize()?alt=java.lang.Object.finalize&text=finalize\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"java.lang.Object.getClass()?alt=java.lang.Object.getClass&text=getClass\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"java.lang.Object.hashCode()?alt=java.lang.Object.hashCode&text=hashCode\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"java.lang.Object.notify()?alt=java.lang.Object.notify&text=notify\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"java.lang.Object.notifyAll()?alt=java.lang.Object.notifyAll&text=notifyAll\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"java.lang.Object.wait()?alt=java.lang.Object.wait&text=wait\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"java.lang.Object.wait(long)?alt=java.lang.Object.wait&text=wait\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"java.lang.Object.wait(long,int)?alt=java.lang.Object.wait&text=wait\" data-throw-if-not-resolved=\"False\" />"
- classRef: "<xref href=\"java.lang.Throwable?alt=java.lang.Throwable&text=Throwable\" data-throw-if-not-resolved=\"False\" />"
  methodsRef:
  - "<xref href=\"java.lang.Throwable.addSuppressed(java.lang.Throwable)?alt=java.lang.Throwable.addSuppressed&text=addSuppressed\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"java.lang.Throwable.fillInStackTrace()?alt=java.lang.Throwable.fillInStackTrace&text=fillInStackTrace\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"java.lang.Throwable.getCause()?alt=java.lang.Throwable.getCause&text=getCause\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"java.lang.Throwable.getLocalizedMessage()?alt=java.lang.Throwable.getLocalizedMessage&text=getLocalizedMessage\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"java.lang.Throwable.getMessage()?alt=java.lang.Throwable.getMessage&text=getMessage\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"java.lang.Throwable.getStackTrace()?alt=java.lang.Throwable.getStackTrace&text=getStackTrace\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"java.lang.Throwable.getSuppressed()?alt=java.lang.Throwable.getSuppressed&text=getSuppressed\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"java.lang.Throwable.initCause(java.lang.Throwable)?alt=java.lang.Throwable.initCause&text=initCause\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"java.lang.Throwable.printStackTrace()?alt=java.lang.Throwable.printStackTrace&text=printStackTrace\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"java.lang.Throwable.printStackTrace(java.io.PrintStream)?alt=java.lang.Throwable.printStackTrace&text=printStackTrace\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"java.lang.Throwable.printStackTrace(java.io.PrintWriter)?alt=java.lang.Throwable.printStackTrace&text=printStackTrace\" data-throw-if-not-resolved=\"False\" />"
  - "<xref href=\"java.lang.Throwable.setStackTrace(java.lang.StackTraceElement[])?alt=java.lang.Throwable.setStackTrace&text=setStackTrace\" data-throw-if-not-resolved=\"False\" />"
syntax: "public final class **ErrorInfoException**</br> extends <xref href=\"com.azure.core.management.exception.ManagementException?alt=com.azure.core.management.exception.ManagementException&text=ManagementException\" data-throw-if-not-resolved=\"False\" />"
constructors:
- uid: "com.azure.resourcemanager.databricks.models.ErrorInfoException.ErrorInfoException(java.lang.String,com.azure.core.http.HttpResponse)"
  fullName: "com.azure.resourcemanager.databricks.models.ErrorInfoException.ErrorInfoException(String message, HttpResponse response)"
  name: "ErrorInfoException(String message, HttpResponse response)"
  nameWithType: "ErrorInfoException.ErrorInfoException(String message, HttpResponse response)"
  summary: "Initializes a new instance of the Error<wbr>Info<wbr>Exception class."
  parameters:
  - description: "the exception message or the response content if a message is not available."
    name: "message"
    type: "<xref href=\"java.lang.String?alt=java.lang.String&text=String\" data-throw-if-not-resolved=\"False\" />"
  - description: "the HTTP response."
    name: "response"
    type: "<xref href=\"com.azure.core.http.HttpResponse?alt=com.azure.core.http.HttpResponse&text=HttpResponse\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public ErrorInfoException(String message, HttpResponse response)"
  desc: "Initializes a new instance of the ErrorInfoException class."
- uid: "com.azure.resourcemanager.databricks.models.ErrorInfoException.ErrorInfoException(java.lang.String,com.azure.core.http.HttpResponse,com.azure.resourcemanager.databricks.models.ErrorInfo)"
  fullName: "com.azure.resourcemanager.databricks.models.ErrorInfoException.ErrorInfoException(String message, HttpResponse response, ErrorInfo value)"
  name: "ErrorInfoException(String message, HttpResponse response, ErrorInfo value)"
  nameWithType: "ErrorInfoException.ErrorInfoException(String message, HttpResponse response, ErrorInfo value)"
  summary: "Initializes a new instance of the Error<wbr>Info<wbr>Exception class."
  parameters:
  - description: "the exception message or the response content if a message is not available."
    name: "message"
    type: "<xref href=\"java.lang.String?alt=java.lang.String&text=String\" data-throw-if-not-resolved=\"False\" />"
  - description: "the HTTP response."
    name: "response"
    type: "<xref href=\"com.azure.core.http.HttpResponse?alt=com.azure.core.http.HttpResponse&text=HttpResponse\" data-throw-if-not-resolved=\"False\" />"
  - description: "the deserialized response value."
    name: "value"
    type: "<xref href=\"com.azure.resourcemanager.databricks.models.ErrorInfo?alt=com.azure.resourcemanager.databricks.models.ErrorInfo&text=ErrorInfo\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public ErrorInfoException(String message, HttpResponse response, ErrorInfo value)"
  desc: "Initializes a new instance of the ErrorInfoException class."
methods:
- uid: "com.azure.resourcemanager.databricks.models.ErrorInfoException.getValue()"
  fullName: "com.azure.resourcemanager.databricks.models.ErrorInfoException.getValue()"
  name: "getValue()"
  nameWithType: "ErrorInfoException.getValue()"
  overridden: "com.azure.core.management.exception.ManagementException.getValue()"
  syntax: "public ErrorInfo getValue()"
  returns:
    type: "<xref href=\"com.azure.resourcemanager.databricks.models.ErrorInfo?alt=com.azure.resourcemanager.databricks.models.ErrorInfo&text=ErrorInfo\" data-throw-if-not-resolved=\"False\" />"
type: "class"
desc: "Exception thrown for an invalid response with ErrorInfo information."
metadata: {}
package: "com.azure.resourcemanager.databricks.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-databricks:1.0.0
